### ISSUE TITLE
Launcher wont load correctly - jsons not loading

### ISSUE THREAD

**Issue by RyanTheAllmighty** [ATLauncher/ATLauncher#631]
There is an issue which seems to happen every now and then for people (all who are on Java 17 Adoptium) where the launcher will not load correctly, the Minecraft versions json file doesn't get red, so a bunch of `No Minecraft version found matching` comes up.

No errors show in the logs, but deleting the `configs/json` folder for the launcher solves it.

Since no errors come up, it's hard to trace back, but according to sentry, all the errors only come from Eclipse Adoptium 17.0.3 and only occurs on Windows (so far anyway).

No idea what the cause it, just what the workaround is. Even if the json files were corrupt, the launcher should be redownloading it correctly anyway, so it's an odd one

**Comment by RyanTheAllmighty** [comment]
An attempted fix has been pushed with 8ee0885e99ddf7d5fd11ed98f3e8a88e5e856aa6 will need to monitor if that helps the issue

**Comment by RyanTheAllmighty** [comment]
No dice, fix doesn't seem to work. Previous thoughts of it only showing up on Java 17 were incorrect, the grouping in Sentry was just not including all events. Seems to be across all Java versions and operating systems.

**Comment by Obscure2020** [comment]
Thanks for confirming that.
As I'm sure you remember, I was one of the people who experienced this bug.

I thought I might be an outlier, though, because I definitely did _not_ get my Java 17 from Adoptium.
I got it from here instead: [https://jdk.java.net/17/](https://jdk.java.net/17/) (The download links are now gone, but they used to be there. That was the version of Java I was running at the time I experienced this glitch.)

Glad to know that this isn't a Java-version or Java-provider or system-dependent bug. That increases the chances this is a code bug that could actually be found and fixed.
